JDI.Light/JDI.Light/Extensions/ObjectExtensions.cs

```diff
@@ -27,6 +27,7 @@ public static class ObjectExtensions
                     case IConvertible _:
                         strValue = v.ToString();
                         break;
+                    default: break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'default' clause.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5599701)